shops/management/commands/import_weekly_data.py

```diff
@@ -18,19 +22,36 @@ def add_arguments(selef, parser):
 
 
     def handle(self, *args, **kwargs):
-        delete_data = input('You are about to delete all existing weekly data, are you sure? (YES/Y to confirm, any other key to quit)')
+        delete_data = input('You are about to delete all existing weekly data, \
+            are you sure? (YES/Y to confirm, any other key to quit)')
         if delete_data.lower() not in ['y', 'yes']:
             pass
         else:
             delete_all_objects()
-            data_path = './data/weekly_shop_data'
+            data_path = './data/weekly_shop_'
             for weekly_data in pathlib.Path(data_path).iterdir():
-                fruit_data, overhead_data, shop_code, date_object = import_data(weekly_data)
+                fruit_data, overhead_data, shop_code, date_object \
+                    = import_helpers.import_data(weekly_data)
                 shop = models.Shop.objects.get(code=shop_code)
-                weekly_shop_summary = models.WeeklyShopSummary(shop=shop, date=date_object)
+                weekly_shop_summary = models.WeeklyShopSummary(
+                    shop=shop, 
+                    date=date_object
+                    )
+                weekly_shop_summary.upload = File(
+                    file=open(weekly_data, 'rb'),
+                     name=Path(weekly_data).name
+                     )
```

Comment:
You could do this as a single statement:
weekly_shop_summary = models.WeeklyShopSummary(
                    shop=shop, 
                    date=date_object,
upload = File(
                    file=open(weekly_data, 'rb'),
                     name=Path(weekly_data).name
                     )                    
)

(but add some better indentation - GitHub comments won't let me do indentation)